.github/workflows/windows_ci.yml

```diff
@@ -31,8 +31,17 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: ${{ matrix.architecture }}
       
-      - name: Install dependencies and test ONNX models
+      - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install onnx==${{ matrix.onnx-version }} requests
-          python .azure-pipelines/test_models.py
\ No newline at end of file
+          python -m pip install onnxruntime
+          git clone https://github.com/microsoft/onnxruntime.git
+      - name: Test new models by onnx
+        run: |
+          $Env:PYTHONPATH=".\onnxruntime\tools\python";
```

Comment:
I cannot find a good way to set global env variables in Winodws-CI properly...  Use two `$Env:PYTHONPATH=".\onnxruntime\tools\python";` for now.

Reply:
Did you check this : https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables

Reply:
Updated. Thanks for the info.